Java-FlightServer/src/flight-client/main/java/flight_client/org/m4d/adp/flight_client/RunBenchmark.java

```diff
@@ -0,0 +1,152 @@
+package org.m4d.adp.flight_client;
+
+import org.m4d.adp.flight_server.ExampleFlightServer;
+import org.m4d.adp.flight_server.ExampleProducer;
+import org.m4d.adp.flight_server.RelayProducer;
+import org.apache.arrow.flight.*;
+import org.apache.arrow.memory.BufferAllocator;
+import org.apache.arrow.memory.RootAllocator;
+import org.apache.arrow.util.AutoCloseables;
+import org.apache.arrow.vector.VectorSchemaRoot;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+public class RunBenchmark {
```

Comment:
Please add comments describing what it is you are testing and how you are testing it.  In general some documentation for all the functions would be helpful